test/e2e/query_test.go

```diff
@@ -2376,93 +2376,3 @@ func TestDistributedEngineWithExtendedFunctions(t *testing.T) {
 	}, time.Now, promclient.QueryOptions{}, 1)
 	testutil.Equals(t, model.SampleValue(0), result[0].Value)
 }
-
-func TestDistributedEngineWithDisjointTSDBs(t *testing.T) {
-	t.Skip("skipping test as this replicates a bug")
```

Comment:
This got converted to a passing test right? Perhaps we could have one case where the test passes and another where it fails?

Reply:
@fpetkovski added a failing test, which is flaky.

Happy flow with flag enabled is successful in ~10 seconds.

Unhappy flow with the disabled flag can be successful in around 40~ seconds. I haven't validated it yet, but I suspect its because an endpoint is not taken into account and we don't converge on high MinT